### ISSUE TITLE
How to remove `gui` styling from highlight groups?

### ISSUE THREAD

**Issue by gegoune** [akinsho/bufferline.nvim#431]
Hey Akin, not really an issue per se as much as change most likely coming from migration to `nvim_set_hl()`.

I have previously removed `italic` and `bold` from various `_selected` highlight groups by assigning `none` to `gui` parameter:

```lua
require('bufferline').setup {
  highlights = {
    buffer_selected = { gui = 'none' },
    ...
  }
}
```

It used to work, but unfortunately now errors with: `Failed setting BufferLineBufferSelected  highlight, something isn't configured correctly: invalid key: none` which indicates that `nvim_set_hl()` doesn't accept that value? I can change `none` to `bold` which works and removes `italic` but only partially solves my problem of also not wanting `bold` in there.

Would you know how could I blank out `gui` for that highlight group? Have tried `None`, `none` and `NONE`.

Thanks!

**Comment by akinsho** [comment]
Hey @gegoune,

So I just realized you can't technically, the issue is that I'm still converting highlights from the old keys for the `highlight` command to arguments for `nvim_set_hl` so as not to break a bunch of people's configs. I'd really like to switch to using just the `nvim_set_hl` keys but don't want to break things for people.

I think what I'll do is allow specifying the `nvim_set_hl` keys as well as the old keys and add a deprecation notice in the future, so people start using those, and I can just remove that code.

**Comment by akinsho** [comment]
Can you try #432 I haven't really tested it, but hopefully you can directly set values like `bold = false` etc. in your config

**Comment by gegoune** [comment]
Thanks for that, unfortunately doesn't seem to work:

```lua
require('bufferline').setup {
  highlights = {
    buffer_selected = { bold = false, italic = false },
  },
},
```

still creates highlight group:
```
BufferLineBufferSelected xxx cterm=bold,italic gui=bold,italic guifg=#cdcecf guibg=#192330
```

I also tried:
```lua
    buffer_selected = { gui = { bold = false, italic = false } },
```
but it prints notice at start:
```
removing buffer_selected as it is not formatted correctly
```

**Comment by akinsho** [comment]
Okay, you should now be able to just do `gui=NONE` or `gui=none`

**Comment by gegoune** [comment]
It's working nicely again, thank you very much!

**Comment by akinsho** [comment]
closed by #432
